docs/SMCWG-charter.md

```diff
@@ -0,0 +1,334 @@
+Mail List or provide evidence to the SMCWG Chair who SHALL post it to the SMCWG
+Public Mail List. The SMCWG Chair will examine the evidence and unsuspend the
+member, or not, by announcement to the SMCWG Public Mail List. A Member’s
+membership will automatically cease six months after the SMCWG Chair’s
+announcement of its suspension if the Member has not re-met the membership
+criteria by that time.
+
+While suspended, a Member may participate in SMCWG Meetings, SMCWG
+Teleconferences, and on the SMCWG’s discussion lists, but may not propose or
+endorse ballots or take part in any form of voting.
+
+Votes cast before the announcement of a Member’s suspension will stand.
+
+### 5. Membership Declaration
+
+In accordance with the IPR Policy, Members that choose to participate in the
+SMCWG MUST declare their participation and SHALL do so prior to participating.
+The Chair of the SMCWG SHALL establish a list for declarations of participation
+and manage it in accordance with the Bylaws, the IPR Policy, and the IPR
+Agreement.
+
+### 6. Voting and Other Organizational Matters
+
+#### 6.1. Voting Structure
+
+The rules described in Bylaw 2.3 and 2.4 SHALL apply to all ballots, including
+Draft Guideline Ballots.
+
+In order for a ballot to be adopted by the SMCWG, two-thirds or more of the
+votes cast by the Certificate Issuers must be in favor of the ballot and more
+than 50% of the votes cast by the Certificate Consumers must be in favor of the
+ballot. At least one member of each class must vote in favor of a ballot for it
+to be adopted. Quorum is the average number of Member organizations (cumulative,
+regardless of Class) that have participated in the previous three (3) SMCWG
+Meetings or Teleconferences (not counting subcommittee meetings thereof). No
+Ballots shall be adopted until at least (3) meetings have occurred and quorum
+determined.
+
+#### 6.2. Other Organizational Matters
+
+- The Chair MAY delegate any of their duties to the Vice Chair as necessary.
+  The Vice Chair has the authority of the Chair in the event of any absence or
+  unavailability of the Chair, and in such circumstances, any duty delegated
+  to the Chair herein MAY be performed by the Vice Chair. For example, the Vice
+  Chair MAY preside at SMCWG Meetings and Teleconferences in the Chair’s
+  absence.
+- The SMCWG may establish subcommittees, by formal ballot, to address any of the
+  SMCWG’s business as specified in the ballot (each, a “SMCWG Subcommittee”).
+  SMCWG Subcommittees are open to all SMCWG Members. A SMCWG Subcommittee may
+  work on and recommend ballots, complete delegated functions, or issue reports
+  to the SMCWG that are within the scope of this charter. SMCWG Subcommittees
+  must post all agendas and minutes on a public mail list.
+
+### 7. Summary of Major Deliverables
+
+The primary deliverable of the SMCWG is set forth in the Scope section, and
+shall include a Draft Guideline be named the "Baseline Requirements for S/MIME
```

Comment:
Consider naming the deliverable "Baseline Requirements for the Issuance and Management of Publicly-Trusted S/MIME (or Email Protection) Certificates" to align with the naming convention used by the BRs and CSBRs.

Reply:
@timfromdigicert You can make edits depending on how you feel here.

Reply:
I think alignment is good.  Agree? @wthayer @benwilsonusa @clintwilson

Reply:
I'm good with Corey's suggested edit, though don't feel strongly about it.

Reply:
Yes, I accept this change.

Reply:
Fixed in 2aa376c06b45146249d0cc6b8cc5d42d08ccb177